canonical-websites/www.ubuntu.com
# K8s docs updates

## Done

- based on #4391
- updated docs with cleaned up markdown
- initial pass at design updates
- added prism syntax highlighting
- added x scrolling for code blocks

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/kubernetes/docs](http://0.0.0.0:8001/kubernetes/docs)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)

## Issue / Card

Fixes #4344
